src/server/accessibility/__snapshots__/index.test.js.snap

```diff
@@ -0,0 +1,53 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`#accessibilityStatementPage should render expected response and content 1`] = `
+"
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <div class="govuk-body" data-testid="app-page-body">
```

Comment:
I can't see this extra wrapper anywhere in https://design-system.service.gov.uk/styles/layout/ - should we remove?  and move the `data-testid` up to the element above?

Reply:
everywhere? or just this file?

Reply:
ahh - we've done this everywhere?  let's not worry about that in this ticket, then :)